src/ccmain/control.cpp

```diff
@@ -1423,9 +1422,8 @@ void Tesseract::classify_word_pass1(const WordData& word_data,
                                     classify_bln_numeric_mode,
                                     textord_use_cjk_fp_model,
                                     poly_allow_detailed_fx, row, block);
-#endif  // ndef DISABLED_LEGACY_ENGINE
+  #endif  // ndef DISABLED_LEGACY_ENGINE
```

Comment:
```suggestion
#endif  // ndef DISABLED_LEGACY_ENGINE
```

Reply:
@stweil This is intentional change. Look at the indent of opening macro at line 1419/1418.